src/jmc/compile/command/utils.py

```diff
@@ -371,8 +371,9 @@ def eval_expr(expr: str) -> str:
 
 
 OPERATORS: dict[type, Callable[..., Any]] = {ast.Add: op.add, ast.Sub: op.sub, ast.Mult: op.mul,
-                                             ast.Div: op.truediv, ast.Pow: op.pow,
-                                             ast.USub: op.neg}
+                                             ast.Div: op.truediv, ast.FloorDiv: op.floordiv, ast.Mod: op.mod, ast.Pow: op.pow, 
+                                             ast.BitAnd: op.and_, ast.BitOr: op.or_, ast.BitXor: op.xor,
```

Comment:
unnecessary operatrors
`ast.BitAnd: op.and_, ast.BitOr: op.or_, ast.BitXor: op.xor,`

Reply:
yeah I guess you can probably remove those; floor division & modulo support were the important ones